### DISCUSSION TITLE
Welcome to sweetviz Discussions!

### DISCUSSION THREAD

**Discussion by fbdesignpro** [fbdesignpro/sweetviz#69] [General]
## 👋 Let's talk Sweetviz and EDA!
I'm very excited that GitHub is now supporting forum-style Discussions!

I always wanted Sweetviz to be as useful as possible, and the best way is to hear feedback from you. So don't be shy and please post anything on your mind about Sweetviz and getting better EDA. Cheers!

Francois

**Comment by gunarevuri** [comment]
Hi, Myself interested to integrate sweetviz library to my flask application. do I need any licence to use it in my application?
If Yes. what I have to do for it.

**Comment by fbdesignpro** [comment]
Hello @gunarevuri, thank you for your interest! I'm happy to report that Sweetviz is licensed under an MIT license which pretty much gives you any permission you want, no license required. The only requirement is to mention the license, as specified here:

https://github.com/fbdesignpro/sweetviz/blob/master/LICENSE

Enjoy! :)

**Reply by gunarevuri** [reply]
Thank you @fbdesignpro for a quick response, I don't want to put SweetViz images or logo in my application. Can you please guide me in that?
![sweetviz_logo](https://user-images.githubusercontent.com/39848983/105290787-7b093380-5bde-11eb-84e5-ff03b8ef26c6.png)


Thanks,
Guna Revuri

**Reply by fbdesignpro** [reply]
Hi @gunarevuri ! Yes you can simply use an override INI setting (or change the installed `sweetviz_defaults.ini`).

Check out the section "**Customizing defaults: the Config file**" in the [readme](https://github.com/fbdesignpro/sweetviz).

This should totally work, please let me know if you have any issues!

Thank you,

Francois

**Reply by fbdesignpro** [reply]
Hi @gunarevuri I just wanted to see if that worked. The reason I ask is that it **should** work but some people had issues removing the logo so I want to make sure the system is working. :)

**Comment by akaragul** [comment]
Dear @fbdesignpro, 
I just saw your post on Linkedin and found this link through it.
It is amazing to see people like yourself who are reaching out and helping with this great ideas.
I am looking forward to using your code in my tasks.
Kind Regards,
Ali

**Comment by vinay453** [comment]
This is so awesome, thank  you!  I have a few recommendations - 
1. target_feat - can we plot two lines (e.g. - the actual and the predicted values for each var) 
2. It would be useful to be able to specify the left-y and right-y axis so we see data by three dimensions in total
3. For the target_feat, if we can calculate weighted averaged  and relativities before plotting, that would give more insights,

**Reply by fbdesignpro** [reply]
Hi @vinay453, thank you for the good words and suggestions:
1. This is a good use case. Until there's more formal support, you can probably use the "compare" functionality of sweetviz and it should work! i.e. compare a copy of the same dataset with itself, but replacing the "actual" target column with "predicted" data. If you get around to trying it, let me know how it works!
2. Do you mean using multiple scales for cases when the target data has a really low percentage? This feature is very high on the list. If something different, I would love to hear more.
3. I'm not sure what you mean exactly on that one, sorry! :)

**Comment by harshil-95** [comment]
Hi, Thanks a lot for this amazing tool! I was wondering if there is a way to get an exhaustive list of entire category list we see in sweetviz output where currently there is 15% put in "others" category in the end?
![image](https://user-images.githubusercontent.com/80265383/133140358-272c2b06-7007-4c2f-8f55-4d4c0501af4b.png)

**Comment by claudelahoz** [comment]
Hello,
Excellent contribution,
I am using it but I have a problem and it is that some variables that are numerical are being taken as categories, how can I fix this?

**Reply by hrobinson-mdsol** [reply]
I am having the same problem, despite applying force_num at FeatureConfig.
